server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -984,7 +984,7 @@ synchronized void completeClose(boolean saveState, boolean completeClose) throws
             activeScans.size());
         this.wait(50);
       } catch (InterruptedException e) {
-        log.error(e.toString());
+        log.error("Interrupted waiting to completeClose for extent {}, {}", extent, e, e);
```

Comment:
With a custom message, it's no longer necessary to force the exception.toString to be part of the message. That's mostly a hack to satisfy the logging API when you don't have a custom message to satisfy the API's need for a string as the first parameter. Having the stack trace is sufficient, as it will already display the message.

```suggestion
        log.error("Interrupted waiting to completeClose for extent {}", extent, e);
```